app/assets/src/components/views/bulk_download/ChooseStep.jsx

```diff
@@ -395,10 +416,74 @@ class ChooseStep extends React.Component {
     );
   };
 
-  render() {
-    const { onContinue, selectedSampleIds } = this.props;
+  renderInvalidSamplesWarning = () => {
+    const { invalidSampleNames } = this.props;
 
-    const numSamples = selectedSampleIds.size;
+    const header = (
+      <div className={cs.header}>
+        <span className={cs.highlight}>
+          {invalidSampleNames.length} sample
+          {invalidSampleNames.length > 1 ? "s" : ""} won't be included in the
+          bulk download
+        </span>, because they are in progress or failed samples:
+      </div>
+    );
+
+    const content = (
+      <div className={cs.messageContainer}>
+        {invalidSampleNames.map((name, index) => {
+          return (
+            <div key={index} className={cs.messageLine}>
+              {name}
+            </div>
+          );
+        })}
+      </div>
+    );
+
+    return (
+      <CompactListNotification
+        header={header}
+        content={content}
+        open={false}
+        type={"warn"}
+        displayStyle={"flat"}
+      />
+    );
+  };
+
+  renderValidationError = () => {
+    const { validationError } = this.props;
+
+    const header = (
+      <div className={cs.header}>There was an error fetching sample data:</div>
```

Comment:
I would reword this to "an unexpected error occurred when verifying your selected samples" and not show the `validationError`. I think a phrase like "fetching sample data" would be confusing to the user, who has no idea we're   doing all this validation behind-the-scenes.

Reply:
My thinking on this is that showing the error would be useful, since if `validationError` does have a value it's either a message indicating that the user selected more samples than are allowed in a bulk download - a limit that as far as I can tell they have no other way of knowing about - or something more pernicious has occured, and it asks the user to contact us for more help. Seems pretty valuable to me.

Agreed on the rewording.